src/modules/mavlink/mavlink_main.cpp

```diff
@@ -816,38 +816,33 @@ int Mavlink::mavlink_open_uart(int baud, const char *uart_name)
 		return -1;
 	}
 
-	if (!_is_usb_uart) {
```

Comment:
Could you explain why this isn't needed anymore?

Reply:
because this check already exists in Mavlink::enable_flow_control